### PR TITLE
[OSJC-231] Update Build and ReplicationController to

### PR DESCRIPTION
**Pull request by jcantrill**
[OSJC-231] Update Build and ReplicationController to support changes in JBT explorer for deployments

### CODE REVIEW

**Comment by jcantrill** [comment]
[test] cc @fbricon @adietish

File: src/main/java/com/openshift/restclient/model/IDeploymentConfig.java [hunk 1]
```diff
@@ -26,6 +27,14 @@
 	Collection<String> getTriggerTypes();
 
 	/**
+	 * Returns the trigger of the given type
+	 * or null if it doesn not exist
+	 * @param type
+	 * @return
+	 */
+	List<IDeploymentTrigger> getTriggers();
```
Comment (fbricon) [inline src/main/java/com/openshift/restclient/model/IDeploymentConfig.java]: why a list, not a collection as above?

**Comment by jcantrill** [comment]
[test]

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**